src/site/content/en/blog/more-capable-form-controls/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: "More capable form controls"
+subhead: Build custom elements that work like native form controls 
```

Comment:
I guess the problem now is it doesn't make it clear that formdata is also mentioned 😅

What about something like... "With new events, and custom elements APIs, participating in forms just got a lot easier."

Reply:
👍 👍 
Tried to work both in w/o luck. You win!